src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -846,9 +846,9 @@ def _normalize_sku(sku):
 
 def _get_sku_name(tier):
     tier = tier.upper()
-    if tier == 'F1':
+    if tier in ['F1', 'FREE']:
```

Comment:
Ideally, the tier collections should be provided in the SDK or through other sources of truth instead of hardcoded in the CLI.

Reply:
Ideally yes.

Reply:
Is there a github issue in SDK repo tracking this problem?